app/views/items/index.html.erb

```diff
@@ -18,10 +19,10 @@
   <tbody>
     <% @items.each do |item| %>
       <tr>
-        <td><%= item.location.name if item.location %></td>
         <td><%= link_to item.name, item_path(item) %></td>
         <td><%= "#{item.brand} #{item.model}" %></td>
         <td><%= item.category %></td>
+        <td><%= item.location_name %></td>
```

Comment:
this still feels a little uncomfortable to me but i don't currently have a better suggestion. 

It's not just that it knows about an internal thing of location it's just the concept that an item has any clue about the NAME of the place it's located in. Know where it is, sure. Know the name of the place where it is? ... weird.